libs/ts/config-types/src/node-config/types.ts

```diff
@@ -39,15 +39,22 @@ const PublishingCriteriaSchema = S.Struct({
   peg_tolerance_percentage: S.Number,
 });
 
+const ContractSchema = S.Struct({
+  name: S.String,
+  address: S.NullishOr(S.String),
+  creation_byte_code: S.NullishOr(S.String),
+  deployed_byte_code: S.NullishOr(S.String),
+  min_quorum: S.NullishOr(S.Number),
+  contract_version: S.Number,
```

Comment:
Should we keep the support of the old version of our contracts?

Reply:
Maybe, they can be removed in some other PR. I don't want to do it here, because it will take more work to rewrite all existing test to use new contracts